README.md

```diff
@@ -1,6 +1,132 @@
 # fullstory-segment-middleware-android
 
 ## Note: This project is currently under development. 
-FullStory Segment Middleware is a project demonstrating the full potential of joining forces between Segment + FullStory.
+--- 
```

Comment:
Please remove the line separator - they introduce visual artifacts:

![image](https://user-images.githubusercontent.com/45576380/97488366-8842b900-1934-11eb-81d8-aea8d6e926de.png)